Documentation/ColumnsConfig/Type/Category/Index.rst

```diff
@@ -24,6 +24,7 @@ The following options can be overridden via :ref:`page TSconfig, TCE form
 <t3tsconfig:pageTsConfigTceFormConfig>`:
 
 *  `size`
+*  `foreign_table_where`
```

Comment:
This seems not to be correct: https://github.com/TYPO3/typo3/blob/main/typo3/sysext/backend/Classes/Form/Utility/FormEngineUtility.php#L59

Reply:
What you are referencing is the ability to overwrite TCA in the TSconfig file. What this documentation is talking about is overwriting TCA presets in the TCA definition.

Reply:
Sorry, I realize what I said is wrong. It is of course talking about the TSconfig file in the first part, but I was confused because of the sentence in the note section. The fact that the list is inserted between two parts that talk about the TCA file never made me question that introduction sentence. I always skipped reading it apparently. Sorry for the confusion.